### PR TITLE
API-33858-poa-v1-email-phone-optional

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

Made email and phone number not required for POA v1:
POST /forms/2122
POST /forms/2122/validate
Updates documentation to reflect email and phone attributes as optional.

## Related issue(s)

[API-33858](https://jira.devops.va.gov/browse/API-33858)

## Testing done

local swagger

## Screenshots
2122 & 2122/validate
Screenshot 2024-02-02 at 8 27 46 AM
Screenshot 2024-02-02 at 8 27 35 AM

## What areas of the site does it impact?
	modified:   modules/claims_api/app/swagger/claims_api/v1/swagger.json
	modified:   modules/claims_api/config/schemas/v1/2122.json

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/v1/swagger.json [hunk 1]
```diff
@@ -358,7 +430,7 @@
                   "errors": [
                     {
                       "title": "Resource not found",
-                      "detail": "Claims not found",
+                      "detail": "The BGS server did not find the resource.",
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v1/swagger.json:433]: I thought we removed the callout for "BGS server" in the tooling that returned this before? is that not the case?

Comment (stiehlrod) [reply modules/claims_api/app/swagger/claims_api/v1/swagger.json:433]: Yes, you are correct, I forgot about that. I will track down from where this came.

Comment (FonzMP) [reply modules/claims_api/app/swagger/claims_api/v1/swagger.json]: [it's from here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/lib/claims_api/error/soap_error_handler_spec.rb). This spec defines a controller and assigns the routes and then mocks the response from the controller, so we're essentially calling a fully mocked spec.

Comment (stiehlrod) [reply modules/claims_api/app/swagger/claims_api/v1/swagger.json]: yep, I already updated that spec!

**Pull request merged**